apps/dh/api-dh/source/DataHub.WebApi/Startup.cs

```diff
@@ -110,10 +104,10 @@ public void ConfigureServices(IServiceCollection services)
                 });
             }
 
-            var apiClientSettings = Configuration.GetSection("ApiClientSettings").Get<ApiClientSettings>();
-            AddDomainClients(services, apiClientSettings);
+            var apiClientSettings = Configuration.GetSection("ApiClientSettings").Get<ApiClientSettings>()
+                                    ?? new ApiClientSettings();
```

Comment:
I'm not sure this is necessary. Reading the docs "Get<ApiClientSettings>" will return "default(T)" if settings cannot be binded.

Reply:
But it is. Swagger file generation fails if I remove the null coalescing.